SYCL/ESIMD/api/functional/ctors/operator_assignment.hpp

```diff
@@ -0,0 +1,97 @@
+//===-- operator_assignment.hpp - Functions for tests on simd assignment
+//      operators. --------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functios for tests on simd assignment operators.
```

Comment:
```suggestion
/// This file provides functions for tests on simd assignment operators.
```

Reply:
I've done it [here](https://github.com/intel/llvm-test-suite/pull/762/commits/afbf175e94a84d6a621ba620d2c0119abd0ec6aa#diff-c5eea7de2a62b10b21518cace82c8c9c376b703b2d585f6f25b9bfd4ef1ee8a4R11)